### PR TITLE
Fetch observations from spanner graph.

### PR DESCRIPTION
**Pull request by keyurva**
* Will wire it to the datasource in a follow-up PR.

### CODE REVIEW

File: internal/server/spanner/query.go [hunk 1]
```diff
@@ -18,25 +18,86 @@ package spanner
...
+		FROM StatVarObservation
+		WHERE
+			variable_measured IN UNNEST(@variables) AND
+			observation_about IN UNNEST(@entities) AND
+			observation_date = @date AND
+			value != ''
+		ORDER BY observation_date ASC
+	`,
+		getSelectColumns(ObsColumns, "")),
+	getLatestObsByVariableAndEntity: fmt.Sprintf(`
```
Comment (n-h-diaz) [inline internal/server/spanner/query.go:80]: I think this will return the latest observation overall, rather than per facet

Comment (keyurva) [reply internal/server/spanner/query.go:80]: hmm - true. So in that case, we probably fetch them all from the DB and then process in mixer to return the latest per facet?

**Comment by keyurva** [comment]
> How is the performance?

We need to measure it under load but it was acceptable when generating the goldens.

File: internal/server/spanner/query.go (hunk 1 above)
Comment (n-h-diaz) [reply internal/server/spanner/query.go:80]: That could work, or we could group by facet in the query

Comment (keyurva) [reply internal/server/spanner/query.go:80]: Let me try the latter. Will fetch less data.

Comment (keyurva) [reply internal/server/spanner/query.go:80]: Done. Let me know if this looks ok.

**Review by n-h-diaz — APPROVED** [review]
Thanks!

**Comment by keyurva** [comment]
Thanks for the review!

**Pull request merged**